examples/mysql/cluster-orc.yaml

```diff
@@ -0,0 +1,115 @@
+apiVersion: apps.kubeblocks.io/v1alpha1
+kind: Cluster
+metadata:
+  name: mysql-cluster
+  namespace: default
+spec:
+  # Specifies the behavior when a Cluster is deleted.
+  # - `DoNotTerminate`: Prevents deletion of the Cluster. This policy ensures that all resources remain intact.
+  # - `Halt`: Deletes Cluster resources like Pods and Services but retains Persistent Volume Claims (PVCs), allowing for data preservation while stopping other operations.
+  # - `Delete`: Extends the `Halt` policy by also removing PVCs, leading to a thorough cleanup while removing all persistent data.
+  # - `WipeOut`: An aggressive policy that deletes all Cluster resources, including volume snapshots and backups in external storage. This results in complete data removal and should be used cautiously, primarily in non-production environments to avoid irreversible data loss.
+  terminationPolicy: Delete
+  # Defines a set of node affinity scheduling rules for the Cluster's Pods. This field helps control the placement of Pods on nodes within the cluster.
+  affinity:
+   # Specifies the anti-affinity level of Pods within a Component. It determines how pods should be spread across nodes to improve availability and performance.
+    podAntiAffinity: Preferred
+    # Represents the key of node labels used to define the topology domain for Pod anti-affinity and Pod spread constraints. 
+    topologyKeys:
+    - kubernetes.io/hostname
+    # Determines the level of resource isolation between Pods. It can have the following values: `SharedNode` and `DedicatedNode`. 
+    # - SharedNode: Allow that multiple Pods may share the same node, which is the default behavior of K8s. 
+    # - DedicatedNode: Each Pod runs on a dedicated node, ensuring that no two Pods share the same node. 
+    tenancy: SharedNode
```

Comment:
The `tenancy` field is deprecated and should not appear in example

Reply:
I will create a new pr to adapt the new api for all engines